src/SourceEngine.Demo.Parser/SourceEngine.Demo.Parser.csproj

```diff
@@ -35,6 +35,11 @@
         <DebugSymbols>true</DebugSymbols>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+      <DebugType>full</DebugType>
+      <DebugSymbols>true</DebugSymbols>
+    </PropertyGroup>
```

Comment:
I don't think this should be used or exist. Release builds have the `YOLO` constant set and its debug configuration is _YoloDebug_. `YOLO` just means that `UnsafeBitStream` is used instead of `ManagedBitStream`.